reference/3.0/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md

```diff
@@ -9,178 +9,252 @@ title:  about_PowerShell_exe
 
 ## SHORT DESCRIPTION
 
-Explains how to use the PowerShell.exe command-line tool. Displays the syntax
-and describes the command-line switches.
+Explains how to use the **PowerShell.exe** command-line tool. Displays the
+syntax and describes the command-line switches.
 
-PowerShell.exe starts a Windows PowerShell session. You can use it in Cmd.exe
-and in Windows PowerShell.
+PowerShell.exe starts a PowerShell session. You can use it in
+**Cmd.exe** and in PowerShell.
 
 ## LONG DESCRIPTION
 
 ### SYNTAX
 
 ```
 PowerShell[.exe]
-[-EncodedCommand <Base64EncodedCommand>]
-[-ExecutionPolicy <ExecutionPolicy>]
-[-InputFormat {Text | XML}]
-[-Mta]
-[-NoExit]
-[-NoLogo]
-[-NonInteractive]
-[-NoProfile]
-[-OutputFormat {Text | XML}]
-[-PSConsoleFile <FilePath> | -Version <Windows PowerShell version>]
-[-Sta]
-[-WindowStyle <style>]
-[-File <FilePath> [<Args>]]
-[-Command { - | <script-block> [-args <arg-array>]
-| <string> [<CommandParameters>] } ]
+    [-PSConsoleFile <file> | -Version <version>]
+    [-NoLogo]
+    [-NoExit]
+    [-Sta]
+    [-Mta]
+    [-NoProfile]
+    [-NonInteractive]
+    [-InputFormat {Text | XML}]
+    [-OutputFormat {Text | XML}]
+    [-WindowStyle <style>]
+    [-EncodedCommand <Base64EncodedCommand>]
+    [-ConfigurationName <string>]
+    [-File - | <filePath> <args>]
+    [-ExecutionPolicy <ExecutionPolicy>]
+    [-Command - | { <script-block> [-args <arg-array>] }
+                | { <string> [<CommandParameters>] } ]
```

Comment:
I wonder why the list is not ABC sorted. Descriptions below too.

Reply:
This is sorted in the order presented by the `-?` help output.

Reply:
```
 pwsh -?

Usage: pwsh[.exe] [[-File] <filePath> [args]]
                  [-Command { - | <script-block> [-args <arg-array>]
                                | <string> [<CommandParameters>] } ]
                  [-ConfigurationName <string>] [-CustomPipeName <string>]
                  [-EncodedCommand <Base64EncodedCommand>]
                  [-ExecutionPolicy <ExecutionPolicy>] [-InputFormat {Text | XML}]
                  [-Interactive] [-NoExit] [-NoLogo] [-NonInteractive] [-NoProfile]
                  [-OutputFormat {Text | XML}] [-Version] [-WindowStyle <style>]
                  [-WorkingDirectory <directoryPath>]

       pwsh[.exe] -h | -Help | -? | /?

PowerShell Online Help https://aka.ms/pscore6-docs

All parameters are case-insensitive.

-Command | -c
```

It is ABC sorted (exception is "-File")

Reply:
In any case ABC order is more user friendly.

Reply:
You are commenting about the v3 version of the the article which is for powershell.exe and not pwsh. The v6 version is for pwsh.

Reply:
Sorry, I have never before looked at how the directory structure is organized.

Nevertheless, even for powershell.exe, sorted descriptions will be better, I think.